woocommerce/woocommerce-android
# Display product attributes in order detail

Closes #1286 by displaying the attributes of a purchased product in the order detail view. This includes the following changes:

- Display a comma-separated list of the selected variable product attributes if available and separate that information from the quantity section with a bullet (•). If there are no attributes to display, don't display anything including the bullet.

~**[This FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/issues/1725) and [this FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1740) must be merged first and this PR updated to point to that version before this PR can be reviewed.**~

### Design
![image](https://user-images.githubusercontent.com/5810477/96793667-cbe47280-13ca-11eb-96b3-b439c2101654.png)

### Screenshots

Before | After 
-- | --
<img src="https://user-images.githubusercontent.com/5810477/96793709-dd2d7f00-13ca-11eb-8b7e-34778a18d77e.png" width="350"/>|<img src="https://user-images.githubusercontent.com/5810477/97508678-986b9000-1956-11eb-96e6-f84989948f0b.png" width="350"/>


### To Test

**Important**: Be sure to test on a store that is running the latest beta version of WooCommerce (4.7.0-rc-1) or higher, as well as a store that is running an older version to ensure the changes are backwards compatible. 
1. Create a new order that contains at least one variable item and one non-variable item. 
2. Open the order in the app and verify attributes of the variable product are displayed as designed. Also verify the product name no longer includes variation details.
3. Verify the non-variable product is displayed properly as well (there should be no bullet (•)). 

Update release notes:
